game/src/sandbox/gameplay/commute.rs

```diff
@@ -197,19 +197,17 @@ impl GameplayState for OptimizeCommute {
         self.meter.draw(g);
     }
 
-    fn recreate_panels(&mut self, ctx: &mut EventCtx, app: &App) {
+    fn recreate_panels(&mut self, ctx: &mut EventCtx, _app: &App) {
         self.top_center = Panel::new(Widget::col(vec![
             challenge_header(ctx, "Optimize the VIP's commute"),
             Widget::row(vec![
                 format!("Speed up the VIP's trips by {}", self.goal)
                     .draw_text(ctx)
                     .centered_vert(),
-                Btn::svg(
-                    "system/assets/tools/hint.svg",
-                    RewriteColor::Change(Color::WHITE, app.cs.hovering),
-                )
-                .build(ctx, "hint", None)
-                .align_right(),
+                ctx.style()
```

Comment:
![Screenshot from 2021-01-21 11-18-17](https://user-images.githubusercontent.com/1664407/105400906-61fe8380-5bda-11eb-8e22-0843d8a649af.png)
Auto resizing gone awry? :P

Reply:
Shoot, I missed some buttons with baked in text. I've replaced those two, and a couple others I found while auditing. I think I've got them all now.

<img width="640" alt="Screen Shot 2021-01-21 at 3 00 09 PM" src="https://user-images.githubusercontent.com/217057/105411725-5ec6c000-5bf9-11eb-8d54-b78fca17024c.png">